charts/argocd-image-updater/values.yaml

```diff
@@ -72,6 +72,14 @@ config:
     #   prefix: quay.io
     #   credentials: env:REGISTRY_SECRET
 
+# whether to create an authentication script, if enabled, an authentication script will be mounted on /script/auth.sh that can be used to authenticate with registries (ECR)
+# refer to https://argocd-image-updater.readthedocs.io/en/stable/configuration/registries/#specifying-credentials-for-accessing-container-registries for more info
+authScript:
+  enabled: false
+  script: |
+    #!/bin/sh
+    echo "add script here"
```

Comment:
Echoing the previous comment, instead of having a single `script` field and an `enabled` flag, maybe we can have a map of file name and content like so

```yaml
authScripts:
  auth1.sh: |
    #!/bin/sh
    echo "auth script 1 here"
  auth2.sh: |
    #!/bin/sh
    echo "auth script 2 here"
```